src/content/self-hosted/manage/troubleshooting.mdx

```diff
@@ -5,6 +5,45 @@ sidebar_label: Troubleshooting
 id: troubleshooting
 ---
 
+## How to extract logs from Okteto when asking for help
+
+When asking for assistance from Okteto Support, please use the [`okteto logs` command](reference/okteto-cli.mdx#logs) to fetch the logs from your Development Environment.
```

Comment:
Which kind of help is this? if it's for a developer yes, `okteto logs`, but also `kubectl logs`.
If it's for an admin, I would mention https://www.okteto.com/docs/self-hosted/manage/diagnostics/

Reply:
Split this into a section for developers and admins. Updated the developer section to use `okteto doctor`